src/Api/V1/Init.php

```diff
@@ -128,9 +128,10 @@ public static function exceptionAJAXHandler($exception, $output)
     public static function exceptionHandler($exception, $output)
     {
         $output = (string) $output;
+        $debugMail = $this->getContainer()->getParameter('fork.debug_email');
 
         // mail it?
-        if (SPOON_DEBUG_EMAIL != '') {
+        if ($debugMail != '') {
```

Comment:
Hmmm, I don't think these methods are used anymore, since debug mails are now send trough monolog. I guess these exceptionHandlers could be removed.